sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -492,9 +492,15 @@ pkg_postinst() {
 		systemctl --root="${ROOT:-/}" enable "${ENABLED_UNITS[@]}"
 	fi
 
+	if type systemctl &>/dev/null; then
+		if [ ! systemctl --quiet --root="${ROOT:-/}" is-enabled getty@.service ]; then
+			systemctl --root="${ROOT:-/}" enable getty@.service || FAIL=1
+		fi
+	fi
+
 	if [[ -z ${REPLACING_VERSIONS} ]]; then
 		if type systemctl &>/dev/null; then
-			systemctl --root="${ROOT:-/}" enable getty@.service remote-fs.target || FAIL=1
```

Comment:
It looks like `remote-fs.target` should be moved out of `REPLACING_VERSIONS`, too?

A general note for testing: We should also check if the service is still enabled when the rootfs is cleaned and the instance rebooted or when a new rootfs is created with Ignition. This is the case because the service symlinks end up under `/usr`, or?